editor/reducer.js

```diff
@@ -667,6 +679,15 @@ export function metaBoxes( state = defaultMetaBoxState, action ) {
 	}
 }
 
+// Create responsive reducer with lib default breakpoints excluding small where we are currently using 782.
```

Comment:
I don't understand the part about 782, can you explain?

Reply:
I corrected the comment, the comment was valid before we had scss imports (added in parallel in other PR) and the values were hardcoded. But now it is not valid.